components/Cards/index.js

```diff
@@ -18,59 +18,36 @@
 //
 // Create a card for each of the articles and add the card to the DOM.
 //Program execution code begins here
-axios.get("https://lambda-times-backend.herokuapp.com/articles")
-.then(hSuccess)
-.catch(hError)
 
-
-function CardCreator({headline,authorPhoto,authorName}) {
-    const div = document.createElement('div');
-    div.setAttribute('class','card');
-
-    const div1 = document.createElement('div');
-    div1.setAttribute('class','headline');
-    div1.textContent = headline;
-
-    const div2 = document.createElement('div');
-    div2.setAttribute('class','author');
-
-
-    const div2a = document.createElement('div');
-    div2a.setAttribute('class','img-container');
-    const img = document.createElement('img');
-    img.src = authorPhoto;
-
-    div2a.appendChild(img);
-    const span = document.createElement('span');
-    span.textContent = `By ${authorName}`;
-
-    div2.append(div2a,span);
-
-    div.append(div1,div2);
-    console.log(div)
-
-    return div;
-}
-
-function hSuccess(response) {
-    const articles = response.data.articles;
-    console.log(articles)
-    let allArticles = [];
-
-    for(let key in articles) {
-        allArticles = [...allArticles,...articles[key]]
-    }
-
-    let Card;
-    const cardDiv = document.querySelector(".cards-container")
-    allArticles.forEach(article => {
-         Card = CardCreator(article);
-         cardDiv.appendChild(Card);
-    })
-
-}
-
-function hError(e) {
-    console.log(e)
-}
+const cardsContainer = document.querySelector('.cards-container');
+axios.get('https://lambda-times-backend.herokuapp.com/articles')
+	.then(response => {
+		
```

Comment:
Looks like this section need to take the `response` data and the createCard function.